helix-core/src/test/java/org/apache/helix/integration/TestHelixCustomCodeRunner.java

```diff
@@ -127,7 +123,10 @@ public void testCustomCodeRunner() throws Exception {
     for (int i = 0; i < nodeNb; i++) {
       participants[i].syncStop();
     }
-    accessor.removeProperty(keyBuilder.liveInstance("newLiveInstance"));
+    HelixDataAccessor accessor =
+        new ZKHelixDataAccessor(_clusterName, new ZkBaseDataAccessor<>(_gZkClient));
+    Builder keyBuilder = accessor.keyBuilder();
+    accessor.removeProperty(keyBuilder.liveInstance("localhost_1000"));
```

Comment:
Nit: make sure instanceconfigs are deleted at the end of the test. Probably dropCluster() takes care of them? Let's confirm.

Reply:
Yes, instance configs are deleted in dropCluster().